Article/Article.js

```diff
@@ -3,27 +3,33 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = "expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandArticle = this.expandArticle.bind(this);
+    this.expandButton.addEventListener('click', this.expandArticle);
```

Comment:
According to the solution code, .bind() only needs to be used in ES5 and previous versions. I didn't realize this when we spoke yesterday. In lieu of these two lines, you can use:

    this.expandButton.addEventListener('click', () => { this.expandArticle() });